### PR TITLE
MNT: rerender

### LINKED ISSUE #23: @conda-forge-admin, please rerender

### PR DESCRIPTION
**Pull request by conda-forge-linter**
Hi! This is the friendly automated conda-forge-webservice.

I've rerendered the recipe as instructed in #23.

Here's a checklist to do before merging.
- [ ] Bump the build number if needed.

Fixes #23

### CODE REVIEW

File: .ci_support/win_64_cuda_compiler_version10.2.yaml [hunk 1]
```diff
@@ -1,12 +0,0 @@
...
-channel_sources:
-- conda-forge,defaults
-channel_targets:
-- conda-forge main
-cuda_compiler:
-- nvcc
-cuda_compiler_version:
-- '10.2'
-target_platform:
-- win-64
```
Comment (leofang) [inline .ci_support/win_64_cuda_compiler_version10.2.yaml:12]: Removing this file is incorrect...

**Review by leofang — CHANGES_REQUESTED** [review]

**Comment by isuruf** [comment]
@conda-forge-admin, rerender

**Comment by leofang** [comment]
Interesting...Thanks @isuruf. Is there a reason we striped off most of the content in `cuda111_112.yaml`?

**Comment by isuruf** [comment]
They are not strictly necessary as conda-smithy gets them from conda-forge-pinning. Btw, in the future please don't take my commits without giving credit. (Use the commit as is or mention in the new commit message)

**Pull request closed without merging**